service/db.go

```diff
@@ -418,6 +418,12 @@ func updateUserRow(uuid string, svcDerived *pb.User, dbDerived *pb.User) error {
 		return err
 	}
 
+	svcDerived.FirstName = newFirstName
+	svcDerived.LastName = newLastName
+	svcDerived.Organization = newOrganization
+	svcDerived.Email = newEmail
+	svcDerived.IsVerified = newIsVerified
```

Comment:
What are these for?

Reply:
So i can send back the updated information to chrome, it was tested that it came back as expected in my integration test